### PR TITLE
Adds drone hats for some holidays

### PR DESCRIPTION
**Pull request by 81Denton**
drone_hat is underused for holidays, I added some where they'd fit thematically. Also a few more greetings for saying "hello"-day.

### CODE REVIEW

**Comment by improvedname** [comment]
can you make flowers day pick out of all wearable flowers so you have some nice variation

**Comment by 81Denton** [comment]
Yeah, that would be great (I have no idea how)

**Comment by 81Denton** [comment]
@PKPenguin321 If I go for this, I get an "expected a constant expression" error on the first line:

	drone_hat = pick(	/obj/item/reagent_containers/food/snacks/grown/moonflower, 
						/obj/item/grown/sunflower, 
						/obj/item/reagent_containers/food/snacks/grown/poppy, 
						/obj/item/reagent_containers/food/snacks/grown/harebell, 
						/obj/item/grown/novaflower)

**Comment by PKPenguin321** [comment]
ask coderbus idk

**Comment by Barhandar** [comment]
Pretty sure it'd need to be in Initialize().

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -202,6 +205,7 @@
 	name = "UFO Day"
 	begin_day = 2
 	begin_month = JULY
+	drone_hat = /obj/item/clothing/mask/facehugger/dead
```
Comment (RandomMarine) [inline code/modules/holiday/holidays.dm]: 🔫👮👮👮👮👮👮👮👮
WEE WOO WEE WOO
YOU ARE IN VIOLATION OF DRONE LAW 2 STEP AWAY FROM THE BEING

**Comment by ExcessiveUseOfCobblestone** [comment]
You can't call procs in the object's default vars, you have to set it in initialize (well in this case New)

**Comment by 81Denton** [comment]
Ah shit I forgot the rainbow hat for 4/20
Next time!

**Pull request merged**